eng/docker-compose/keycloak.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:latest
+    environment:
+      - KEYCLOAK_ADMIN=admin
+      - KEYCLOAK_ADMIN_PASSWORD=admin
+    command:
+      - start-dev
+    ports:
+      - "8081:8080"
+    volumes:
+      - ./keycloak_data:/opt/keycloak/data
```

Comment:
Suggestions:

1. line 8, put a specific version & SHA256 hash code instead of `latest`.
2. lines 10 and 11, make it possible to override with environment variable. For example: `KEYCLOAK_ADMIN: ${KEYCLOAK_ADMIN:-admin}` (if the variable is not set, it defaults to the value "admin")
3. Also set a variable for the port mapping on line 15. For default, I would choose something even more different than 8080, so that there is less chance of overlap with some other user interface. For example, choosing 8023 is random and less likely to be used by something else.
4. Finally, for the volume on line 17, setup a Docker volume rather than a local directory. See the `volumes` section in the Postgresql yml file.